python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -64,29 +64,32 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         dtype=str,
         default=['BAD', 'CR', 'INTRP', 'SAT', 'SUSPECT', 'NO_DATA']
     )
+    minimumUnmaskedFraction = pexConfig.Field(
+        doc="Minimum required fraction of unmasked pixels to use star.",
```

Comment:
"Minimum fraction of unmasked pixels required to use star" might be slightly clearer.